package.json

```diff
@@ -12,9 +12,13 @@
     "e2e": "API_ENV=prod jest --rootDir ./tests/e2e ",
     "local": "npm run docker:start && npm run local:api",
     "local:api": "DB_PORT=3002 DB_HOST='127.0.0.1' NODE_ENV=development node app.js",
-    "postinstall": "npm run test",
+    "postinstall": "npm run test && npm run graphql",
     "start": "pm2 start process.json",
-    "test": "eslint ."
+    "test": "eslint .",
+    "validate": "./bin/validation-graphql.sh",
+    "graphql": "npm run graphql:clean && npm run graphql:generate",
+    "graphql:clean": "rm -rf ./api/graphql/__generated__/schema-defs.js",
```

Comment:
I would suggest using [rimraf](https://www.npmjs.com/package/rimraf) for cross-platform compatibility